econml/metalearners.py

```diff
@@ -95,13 +107,14 @@ def marginal_effect(self, X):
         ----------
         X : matrix, shape (m × dₓ)
             Matrix of features for each sample.
+        T1: scaler
```

Comment:
```suggestion
        T: scalar
```

Again, consider allowing arrays for `T`.